tools/engine_tool/lib/src/commands/flags.dart

```diff
@@ -0,0 +1,13 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// ignore_for_file: public_member_api_docs
+
+// Keep this list alphabetized.
+const String allFlag = 'all';
+const String builderFlag = 'builder';
+const String fixFlag = 'fix';
+const String printDiffsFlag = 'print-diffs';
```

Comment:
Could you expand on the value of putting all flags in a single file? I imagine this will grow organically over time until it becomes unwieldy.